tests/test_integrity.py

```diff
@@ -476,6 +477,51 @@ def test_sources(db):
     assert len(t) == 37, f'found {len(t)} sources from {ref}'
 
 
+# importing table 9 for data in test parameters
+from astropy.table import Table
```

Comment:
Any import statements should be at the top of the file, for consistency and clarity.